src/hbase/test-util/mini-cluster.cc

```diff
@@ -34,7 +34,7 @@ JNIEnv *MiniCluster::CreateVM(JavaVM **jvm) {
   char *classpath = getenv("CLASSPATH");
   std::string clspath;
   if (classpath == NULL || strstr(classpath, "-tests.jar") == NULL) {
-    std::string clsPathFilePath("../target/cached_classpath.txt");
+    std::string clsPathFilePath("../../../hbase-build-configuration/target/cached_classpath.txt");
```

Comment:
This is pretty (smile)

Reply:
Yes, it isn't pretty. It sort of depends on being run from within the `hbase-native-client` sub-directory of the hbase project proper as well (which has had `mvn package` run beforehand). This path at least exists though if that has been done unlike the previous value :) I reckon there's a better way to manage it more elegantly though.